pkgs/applications/networking/cluster/terraform-providers/default.nix

```diff
@@ -53,7 +61,10 @@ let
       # mkisofs needed to create ISOs holding cloud-init data,
       # and wrapped to terraform via deecb4c1aab780047d79978c636eeb879dd68630
       libvirt = automated-providers.libvirt.overrideAttrs (_: { propagatedBuildInputs = [ cdrtools ]; });
-      teleport = callPackage ./teleport { };
+      teleport = automated-providers.teleport.overrideAttrs (_: {
+        sourceRoot = "source/terraform";
+        checkInputs = [ teleport ];
```

Comment:
`doCheck` is false for `automated-providers`.

Reply:
I wonder why it's there then. I will check with upstream to see if the provider needs to be wrapped.